packages/adp-tooling/src/types.ts

```diff
@@ -35,21 +35,41 @@ export interface AdpWriterConfig {
     target: AbapTarget;
     ui5?: {
```

Comment:
Could you please use `frameworkUrl` like https://github.com/SAP/open-ux-tools/blob/main/packages/ui5-application-writer/src/types.ts#L43, so that we keep both objects in sync.
Btw, what is the use case here? Is it for openUI5 or is there a different reason? It doesn't seem like an openUI5 use case since the target system is always an ABAP system, or did I miss something?

Reply:
It is about the internal scenario when the snapshot version is selected and points to an internal endpoint

Reply:
Thanks.